scrapers/nserc/scrape_details.sh

```
@@ -0,0 +1,69 @@
+    -o "$output_file"
+}
+
+export -f download_details
+
+# Process all JSON files and extract IDs
+find data/listing -name "nserc_results_*.json" | while read json_file; do
+  echo "Processing $json_file"
+  
+  # Extract the IDs from the JSON file (last element of each array in "aaData")
+# Extract all detail URLs from the JSON file
+detail_urls=($(jq -r '.aaData[][5]' "$json_file"))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)